editoast/src/views/infra/mod.rs

```diff
@@ -350,23 +350,51 @@ async fn get_speed_limit_tags(
     ))
 }
 
+#[derive(Debug, Clone, Deserialize)]
+struct GetVoltagesQueryParams {
+    #[serde(default)]
+    add_modes: bool,
```

Comment:
We can improve the naming of the option. Here is a suggestion:

```suggestion
    include_rolling_stock_modes: bool,
```